website/www/site/content/en/documentation/sdks/java-multi-language-pipelines.md

```diff
@@ -14,11 +17,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Cross-language transforms
+# Java multi-language pipelines quickstart
 
-With the samples on this page we will demonstrate how to create and leverage cross-language pipelines.
+> **Note:** This page is a work in progress. Please see [Multi-language pipelines](https://beam.apache.org/documentation/programming-guide/#multi-language-pipelines) for full documentation.
 
-> **Note:** Please see the [Beam Programming Guide](https://beam.apache.org/documentation/programming-guide/#multi-language-pipelines) for full documentation on cross-language transforms.
+With the samples on this page we will demonstrate how to create and leverage cross-language pipelines.
```

Comment:
How about:

This page demonstrates how to write a Java pipeline that uses a Python cross-language transform.